### ISSUE TITLE
[Enhancement]: Directory structure - extra layer option

### ISSUE THREAD

**Issue by KaiStarkk** [advplyr/audiobookshelf#1501]
### Describe the feature/enhancement

An optional additional layer(/s) in the directory structure would be a huge help.

There aren't many genres of fiction that aren't touched by this. For example here's the most famous fantasy author of the day:

```
Brandon Sanderson
├───Alcatraz
│   ├───Book 1 - Alcatraz Versus The Evil Librarians
│   ├───Book 2 - The Scrivener’s Bones
│   ├───Book 3 - The Knights of Crystallia
│   ├───Book 4 - Alcatraz Versus The Shattered Lens
│   └───Book 5 - Alcatraz Versus The Dark Talent
├───Cosmere
│   ├───1 - Elantris Saga
│   │   ├───1 - Elantris
│   │   ├───2 - The Hope of Elantris
│   │   └───3 - The Emperor's Soul
│   ├───2 - Mistborn Saga
│   │   ├───1 - The Original Trilogy
│   │   │   ├───Book 1 - The Final Empire
│   │   │   ├───Book 2 - The Well of Ascension
│   │   │   └───Book 3 - The Hero of Ages
│   │   ├───2 - Wax and Wayne series
│   │   │   ├───Book 4 - The Alloy Of Law
│   │   │   └───Book 5 - Shadows Of Self
│   │   └───3 - Mistborn [Collected Tales]
│   ├───3 - The Stormlight Archive
│   │   ├───Book 0.5 - Edgedancer
│   │   ├───Book 1 - The Way of Kings
│   │   ├───Book 2 - Words of Radiance
│   │   ├───Book 3 - Oathbringer
│   │   └───Book 4 - Rhythm of War
│   ├───Arcanum Unbounded The Cosmere Collection
│   └───Warbreaker
├───Infinity Blade
│   ├───Book 1 - Infinity Blade Awakening
│   └───Book 2 - Infinity Blade Redemption
├───Legion
│   └───Books 1-3 - Legion The Many Lives of Stephen Leeds
├───Perfect State
├───Rithmatist
├───Skyward
│   ├───Book 1 - Skyward
│   └───Book 2 - Starsight
├───The Reckoners
│   └───Book 1 - Steelheart
└───White Sand
    ├───Volume 1
    ├───Volume 2
    └───Volume 3

```

**Comment by advplyr** [comment]
Can you explain some more what I'm looking at? Are these series with sub-series? How could the scanner know if it is a sub-series?

**Comment by hobesman** [comment]
It's a universe (Cosmere) with series (Mistborn) and sub series (Mistborn Original Trilogy, Wax and Wayne).

**Comment by KaiStarkk** [comment]
Correct - and as for how the scanner could do this, the way that makes most sense to me would be to allow collections to hold series, then have the following:

Four level structure
Author > Collection > Series > Book

Five level structure
Author > Collection > Series > Sub-series > Book

But I might not be using collection correctly there?
Will admit I’m not fully clear on what collection is for. It seems it can only hold books so might be intended for standards like “DNF”, “Currently reading”, “Up next”? If so maybe they could be made built in like “Finished” currently is - (and sync to Goodreads / TheStorygraph shelves)?
________________________________

**Comment by KaiStarkk** [comment]
I've realised this can be merged with #870

**Comment by advplyr** [comment]
Duplicate of #870

thanks
